frontend/src/cards/UnknownsMapCard.tsx

```diff
@@ -117,12 +118,12 @@ function UnknownsMapCardWithKey(props: UnknownsMapCardProps) {
           unknownEthnicities.length === 0
             ? unknownRaces
             : unknownRaces.map((unknownRaceRow, index) => {
-              return unknownRaceRow[metricConfig.metricId] >
-                unknownEthnicities[index][metricConfig.metricId] ||
-                unknownEthnicities[index][metricConfig.metricId] == null
-                ? unknownRaceRow
-                : unknownEthnicities[index];
-            });
+                return unknownRaceRow[metricConfig.metricId] >
+                  unknownEthnicities[index][metricConfig.metricId] ||
+                  unknownEthnicities[index][metricConfig.metricId] == null
+                  ? unknownRaceRow
+                  : unknownEthnicities[index];
```

Comment:
This gets pretty deeply nested - might be worth breaking out and using if statements for clarity